model/Core/Properties/packageVerificationCodeExcludedFile.md

```diff
@@ -12,9 +12,7 @@ The relative file name of a file to be excluded from the
 A relative filename with the root of the package archive or directory
 referencing a file to be excluded from the `PackageVerificationCode`.
 
-In general, every filename is preceded with a `./`, see
-[RFC 3986 Uniform Resource Identifier (URI): Generic Syntax](https://datatracker.ietf.org/doc/rfc3986/)
```

Comment:
Keep RFC please. 

Or defer this change to the next MINOR version update, which is 3.1.